<launch>

    <node pkg="world_cloud_converter" type="cloud_converter_cuda_node" name="" output="screen">
        <param name="ed_left" value="4.62524"/>
        <param name="ed_right" value="-0.32476"/>
        <param name="ed_ceil" value="7.63428"/>
        <param name="ed_floor" value="-0.36572"/>
        <param name="ed_resolution" value="0.2"/>
        <param name="self_half_height" value="0.257525"/>
        <param name="self_half_width" value="0.257525"/>
        <param name="passthrough_z_max" value="2.0"/>
        <param name="passthrough_z_min" value="-0.3"/>
    </node>

</launch>